lib/adf_info.py

```diff
@@ -261,24 +264,28 @@ def __init__(self, config_file, debug=False):
                 #Get climo years for verification or assignment if missing
                 starting_location = Path(cam_hist_locs[case_idx])
                 files_list = sorted(starting_location.glob('*'+hist_str+'.*.nc'))
-                case_climo_yrs = sorted(np.unique([i.stem[-7:-3] for i in files_list]))
+                case_climo_yrs_str = sorted(np.unique([i.stem[-7:-3] for i in files_list]))
+                case_climo_yrs = []
+                for year in case_climo_yrs_str:
+                   case_climo_yrs.append(int(year))
 
                 #Check if start or end year is missing.  If so then just assume it is the
                 #start or end of the entire available model data.
                 if syear is None:
                     print(f"No given start year for {case_name}, using first found year...")
                     syear = int(case_climo_yrs[0])
-                elif str(syear) not in case_climo_yrs:
+                elif (syear) not in case_climo_yrs:
```

Comment:
No need to have the parentheses here:

```suggestion
                elif syear not in case_climo_yrs:
```